Detectors/ITSMFT/ITS/macros/test/CheckClusters.C

```diff
@@ -79,41 +94,100 @@ void CheckClusters(Int_t nEvents = 10, TString mcEngine = "TGeant3") {
       const auto gloC = c.getXYZGloRot(*gman); // convert from tracking to global frame
       auto lab = (clusLabArr->getLabels(nc))[0];
 
+      std::string clusterPattern;
```

Comment:
To avoid the dependency of the compiled classes on current ``#ifdef _ClusterTopology_`` (see my comment on convertClusterToString method), could you please move this convertClusterToString here as a function in the macro, wrapping extraction of the pattern from the cluster by 
``#ifdef _ClusterTopology_`` / ``#endif`` check?